src/util/psbt/macros.rs

```diff
@@ -130,6 +130,30 @@ macro_rules! impl_psbt_insert_pair {
     };
 }
 
+#[cfg_attr(rustfmt, rustfmt_skip)]
+macro_rules! impl_psbt_insert_hash_pair {
```

Comment:
I think we have a need to write a derive crate for this library, since we have tooo many macros all around..
I has nothing to do with this specific PR, just a general thought